shared_model/validators/signable_validator.hpp

```diff
@@ -29,19 +29,22 @@ namespace shared_model {
      public:
       explicit SignableModelValidator(
           FieldValidator &&validator = FieldValidator())
-          : ModelValidator(std::move(validator)) {}
+          : ModelValidator(validator), field_validator_(validator) {}
 
       Answer validate(const Model &model) const {
         auto answer = ModelValidator::validate(model);
         std::string reason_name = "Signature";
         ReasonsGroupType reason(reason_name, GroupedReasons());
-        ModelValidator::field_validator_.validateSignatures(
+        field_validator_.validateSignatures(
             reason, model.signatures(), model.payload());
         if (not reason.second.empty()) {
           answer.addReason(std::move(reason));
         }
         return answer;
       }
+
+     private:
+      FieldValidator field_validator_;
```

Comment:
What was the reason to introduce it?

Reply:
Because if we pass any_block_validator as a ModelValidator, it does not have field_validator inside, so using ModelValidator::field_validator fails to compile.